source/intro.tex

```diff
@@ -12,9 +12,9 @@
 
 \pnum
 \Cpp{} is a general purpose programming language based on the C
-programming language as described in ISO/IEC 9899:2018
-\doccite{Programming languages --- C} (hereinafter referred to as the
-\defnx{C standard}{C!standard}). \Cpp{} provides many facilities
+programming language as described in \IsoC{}
+\doccite{Programming languages --- C}.
```

Comment:
We're not allowed to add a title next to the number. This needs to go.

Reply:
Done.